frontend/src/routes/forget/+page.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+    import '$styles/Auth.pcss';
+    import { enhance } from '$app/forms';
+</script>
+
+<html class="h-full">
+    <div class="auth-page">
+        <div class="container-page">
+        <div class="auth-row">
+            <div class="p-4 sm:p-7">
+            <div class="text-center">
+                <h1 class="auth-title-style">Forgot password?</h1>
+                <p class="auth-subtitle-style">
+                Remember your password?
+                <a class="auth-redirect-button-style" href="/login">
+                    Sign in here
+                </a>
+                </p>
+            </div>
+
+            <div class="mt-5">
+                <!-- Form -->
+                <form method="POST" use:enhance>
+                <div class="grid gap-y-4">
+                    <!-- Form Group -->
+                    <div>
+                    <label for="email" class="auth-input-field-label-style">Email address</label>
+                    <div class="relative">
+                        <input type="email" id="email" name="email" class="auth-input-field-style" required aria-describedby="email-error">
```

Comment:
can you remove `aria-describedby="email-error"` because it doesn't work with preline and we are going to use our own validation.